### PR TITLE
Corrected links to images

### PR DESCRIPTION
**Pull request by joachimmetz**

### CODE REVIEW

**Comment by joachimmetz** [comment]
@rjcolonna PTAL

File: docs/jtag_htc_wildfire_s.md [hunk 1]
```diff
@@ -59,13 +59,13 @@ rubber band in these pictures).
 |                                                                         |                                                                         |
 |-------------------------------------------------------------------------|-------------------------------------------------------------------------|
 | <figure>
- <img src="riff-to-htc-wildfire-s-connection-1.png"
+ <img src="assets/images/riff-to-htc-wildfire-s-connection-1.png"
```
[the remark this answers is not in the document]
Comment (simsong) [reply docs/jtag_htc_wildfire_s.md:62]: Note that the image is in the git repo:
https://github.com/forensicswiki/wiki/blob/main/images/riff-to-htc-wildfire-s-connection-1.png

So it also appears that the images are not being copied to the wiki.

**Review by simsong — CHANGES_REQUESTED** [review]
Images are not appearing at assets/images.

If possible, we should also enable directory listing for assets/images.

**Comment by joachimmetz** [comment]
> Images are not appearing at assets/images.

@simsong thanks for flagging, spoke with @rjcolonna who is planning to roll out a larger fix and documentation for this.

**Comment by rjcolonna** [comment]
Hey @simsong, 

Yea you are right.  We moved the images to "assets/images" from "images".  There are some images that have not been copied over and there are several documents that are broken.  I will move the images and fix the documents.  Ill close this PR and create a new one.

**Pull request closed without merging**